cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/TicketGrantingTicketImpl.java

```diff
@@ -55,16 +57,22 @@
     private Boolean expired = Boolean.FALSE;
 
     /** Service that produced a proxy-granting ticket. */
-    @Column(name="PROXIED_BY", nullable=true)
+    @Lob
+    @Column(name="PROXIED_BY", nullable=true, length = Integer.MAX_VALUE)
     private Service proxiedBy;
 
     /** The services associated to this ticket. */
     @Lob
-    @Column(name="SERVICES_GRANTED_ACCESS_TO", nullable=false, length = 1000000)
+    @Column(name="SERVICES_GRANTED_ACCESS_TO", nullable=false, length = Integer.MAX_VALUE)
     private final HashMap<String, Service> services = new HashMap<>();
 
+    /** The PGTs associated to this ticket. */
+    @Lob
```

Comment:
from what I can tell, in order for `@OneToMany` to work, the reverse `@ManyToOne` field cannot be on a `@MappedSuperclass`. This means we would need to move the parent `ticketGrantingTicket` property from `AbstractTicket` to both `ServiceTicketImpl` and `TicketGrantingTicketImpl`.

I did this locally and it led to a different exception with the `merge(ticket.getGrantingTicket)` line you added here: https://github.com/Jasig/cas/pull/1429/files#diff-4bdbc0e2e88e2d337ea33f88245438a4R81

I'm going to continue down this line of testing this evening, but wanted to update in the interim.